### ISSUE TITLE
No Feedback from device after writing to actuator com-object

### ISSUE THREAD

**Issue by chefkoch009** [knxd/knxd#236]
this is what i mean:

Layer 1 [ 6:server/Server 49.501] Send(011): 29 00 BC D0 71 03 07 01 01 00 80
Layer 6 [ 1:main 49.501] SN: OK
Layer 6 [ 2:main/L 49.501] SN: OK L
Layer 1 [ 6:server/Server 49.501] Send(015): 04 01 21 00 29 00 BC D0 71 03 07 01 01 00 80
Layer 1 [ 6:server/Server 49.501] Send(015): 04 02 21 00 29 00 BC D0 71 03 07 01 01 00 80
Layer 0 [ 6:server/Server 49.501] Send(017): 06 10 05 30 00 11 29 00 BC D0 71 03 07 01 01 00 80
Layer 0 [ 6:server/Server 49.501] Dropped(017): 06 10 05 30 00 11 29 00 BC D0 71 03 07 01 01 00 80
Layer 0 [ 6:server/Server 49.501] Send(021): 06 10 04 20 00 15 04 01 21 00 29 00 BC D0 71 03 07 01 01 00 80
Layer 0 [ 6:server/Server 49.501] Send(021): 06 10 04 20 00 15 04 02 21 00 29 00 BC D0 71 03 07 01 01 00 80
Layer 0 [ 6:server/Server 49.506] Recv(010): 06 10 04 21 00 0A 04 02 21 00
Layer 8 [18:tunnel/1.1.101 49.506] TUNNEL_ACK
Layer 0 [ 6:server/Server 49.558] Recv(010): 06 10 04 21 00 0A 04 01 21 00
Layer 8 [16:tunnel/1.1.100 49.558] TUNNEL_ACK
Layer 0 [12:B.tpuarts/Base 49.565] Read(008): BC 71 6B 3C 08 E3 00 80
Layer 8 [12:B.tpuarts/Base 49.565] ser state 20>21
Layer 8 [12:B.tpuarts/Base 49.565] state: wait > wait_more
Layer 8 [12:B.tpuarts/Base 49.565] ser state 21>21
Layer 8 [12:B.tpuarts/Base 49.565] state: wait_more > wait_more
Layer 8 [12:B.tpuarts/Base 49.566] ser state 21>21
Layer 8 [12:B.tpuarts/Base 49.566] state: wait_more > wait_more
Layer 8 [12:B.tpuarts/Base 49.566] ser state 21>21
Layer 8 [12:B.tpuarts/Base 49.566] state: wait_more > wait_more
Layer 0 [12:B.tpuarts/Base 49.566] SendAck 11
Layer 8 [12:B.tpuarts/Base 49.566] ser state 21>21
Layer 8 [12:B.tpuarts/Base 49.566] state: wait_more > wait_more
Layer 8 [12:B.tpuarts/Base 49.566] ser state 21>21
Layer 8 [12:B.tpuarts/Base 49.566] state: wait_more > wait_more
Layer 8 [12:B.tpuarts/Base 49.566] ser state 21>21
Layer 8 [12:B.tpuarts/Base 49.566] state: wait_more > wait_more
Layer 0 [12:B.tpuarts/Base 49.571] Read(003): 14 01 1B
Layer 8 [12:B.tpuarts/Base 49.571] ser state 21>21
Layer 8 [12:B.tpuarts/Base 49.571] state: wait_more > wait_more
Layer 8 [12:B.tpuarts/Base 49.571] ser state 21>21
Layer 8 [12:B.tpuarts/Base 49.571] state: wait_more > wait_more

do you have an idea for this phenomenon? with the 0.10 eibd version, I don't have this problem.

**Comment by smurfix** [comment]
Isn't there some more log data following this?

Can you post a log of 0.10 for comparison?

**Comment by smurfix** [comment]
… plus, please use logging filters with v0.14. The tracing option doesn't do this any more.

**Comment by chefkoch009** [comment]
To give you the informations, how i have to start the knxd?
I'd Like to do this at first, because i've to rebuild the eibd.

**Comment by smurfix** [comment]
Well, how do you start it now?

**Comment by chefkoch009** [comment]
With this line:
` knxd  -e1.1.99 -t 1023 -E1.1.100:3 -D -T -R -S  -c tpuarts:/dev/ttyKNX1'

**Comment by chefkoch009** [comment]
Here is the log of knxd with 2x switch on an switch of on Groupaddress 2/1/3.

[knxd-no-response.zip](https://github.com/knxd/knxd/files/852464/knxd-no-response.zip)

The other log from eibd takes a while, because I've to compile it again.

**Comment by smurfix** [comment]
Please use
`knxd -B log -e1.1.99 -t 1023 -E1.1.100:3 -D -T -R -S -c -B log -b tpuarts:/dev/ttyKNX1`
with v0.14, as I really don't have the time to manually decode the packets …

**Comment by chefkoch009** [comment]
here is a cut of log with your suggested start options:
[knxd-no-response-2.zip](https://github.com/knxd/knxd/files/852641/knxd-no-response-2.zip)

the groupaddress is 2/1/3 again.

**Comment by smurfix** [comment]
So what do you mean by "no feedback"?
You send to 2/1/3, you get an answer from 1.1.10, addressed to 2/2/2.
I don't see a problem here.

**Comment by chefkoch009** [comment]
I testet a little around with ETS5. I started the knxd with your options. In the ETS I added a Interface with 1.1.101. 
If I now write a "0" to a groupaddress it shows a gray lined entry and source address 0.0.0. But shouldn't it be the 1.1.101? 
It seems that the "package-received" from the device isn't handled. I don't mean the status. I don't know how to better explain the problem.

**Comment by chefkoch009** [comment]
This is the setup for now:
knxd start parameter: knxd -B log -e 1.1.99 -E1.1.100:10 -t 1023 -D -R -T -S -c -t1023 -b tpuarts:/dev/ttyKNX1

The interface is discoverable with ETS5 and while clicking on "Test" it comes a "Ok". If i choose this interface then it says: IP Tunneling interface with phys. host addr: 1.1.99 and phys.addr: 1.1.101.
in the groupmonitor I send a "On" with DPT1.001 and I get in the Groupmonitor a line with src addr 0.0.0 and the knxd log says:
[knxd-no-response-3.zip](https://github.com/knxd/knxd/files/854285/knxd-no-response-3.zip)

i don't know, why there is the "state: wait_more > wait_more"

**Comment by chefkoch009** [comment]
It seems that i don't get the ACK of package i sent to actuator via knxd. The Status i get.

**Comment by smurfix** [comment]
> i don't know, why there is the "state: wait_more > wait_more"
Umm, that is debugging output. You should kindof expect that when you use debug or trace options …

**Comment by dallmair** [comment]
> If I now write a "0" to a groupaddress it shows a gray lined entry and source address 0.0.0. But shouldn't it be the 1.1.101?

Yes. That was fixed in the meantime, IIRC in commit 3fa3794e5bca2ac7ca18793e7aa9a39f2042f8df and released in 0.14.59.

> It seems that i don't get the ACK of package i sent to actuator via knxd. The Status i get.

Correct. When knxd does not have a dedicated busmonitor interface, it cannot receive LL acknowledge frames (LL_ACK, LL_NACK, LL_BUSY) and therefore cannot send them to ETS. Thus, these are displayed as "Not acknowledged" in ETS.

It seems there is nothing more we can do here, so I'm closing this issue now. Please reopen if you're still facing this problem in a current version of knxd.
